syndicate/core/handlers.py

```diff
@@ -492,4 +497,64 @@ def copy_bundle(ctx, bundle_name, src_account_id, src_bucket_region,
     click.echo('Bundle was copied successfully')
 
 
+@syndicate.command(name='warmup')
+@click.option('--bundle_name', nargs=1)
+@click.option('--deploy_name', nargs=1)
+@click.option('--api_gw_id', nargs=1, multiple=True, type=str)
+@click.option('--stage_name', nargs=1, multiple=True, type=str)
+@click.option('--lambda_auth', default=False, is_flag=True)
+@click.option('--header_name', nargs=1)
+@click.option('--header_value', nargs=1)
+@timeit(action_name='warmup')
+def warmup(bundle_name, deploy_name, api_gw_id, stage_name, lambda_auth,
+           header_name, header_value):
+    """
+       Warmups Lambda resources
+       :param bundle_name: name of the bundle
+       :param deploy_name: name of the deploy
+       :param api_gw_id: id of the API Gateway to warmup Lambda function
+       :param stage_name: name of the API Gateway stage
+       :param lambda_auth: used to specify if Lambda operates as API Gateway
+       Lambda Authorizer
+       :param header_name: Lambda authorization header key
+       :param header_value: Lambda authorization header value
+       :return:
```

Comment:
merge the improve-generate-lambda branch here and rewrite the 'warmup' interface by the example of 'deploy' command